README.md

```diff
@@ -39,7 +39,7 @@ For more installation options and troubleshooting tips, see
 Each Electron release provides binaries for macOS, Windows, and Linux.
 
 * macOS (High Sierra and up): Electron provides 64-bit Intel and ARM binaries for macOS. Apple Silicon support was added in Electron 11.
-* Windows (Windows 10 and up): Electron provides `ia32` (`x86`), `x64` (`amd64`), and `arm64` binaries for Windows. Windows on ARM support was added in Electron 5.0.8.
+* Windows (Windows 10 and up): Electron provides `ia32` (`x86`), `x64` (`amd64`), and `arm64` binaries for Windows. Windows on ARM support was added in Electron 5.0.8. Support for Windows 7 and 8 was [removed in Electron 23](some link), in line with Chromium's Windows deprecation policy.
```

Comment:
let's actually put a link in here though.

Reply:
link to the blog post?

Reply:
Also link to Chromium's Windows deprecation policy.

Reply:
I think just the link to the deprecation policy (sorry, I missed this wasn't in the original commit). We're prepping the blog post, but it may be a few days, and it'd be great to have this merged before we cut the 23-x-y branch.